03-Custom-Actions-In-Python/03-Custom-Action-Features.ipynb

```diff
@@ -0,0 +1,317 @@
+{
```

Comment:
<p>How is <code>super().attach(simulation)</code>  different from <code>self._state=simulation.state</code>  ?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/glotzerlab/hoomd-examples/pull/17/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
It isn't but it is more robust since if we add something to the default attach the first won't break.